app/controllers/media_controller.rb

```diff
@@ -19,18 +19,29 @@ def stream
     authorize! :read, @media
     respond_to do |format|
       format.m3u8 do
-        redirect_to @media.to_playlist_url
+        redirect_to "#{@media.to_playlist_url}?token_string=#{media_token}"
```

Comment:
@jkeck  for some reason, this feels a little clunky -- we're doing a GET call and including the params in the redirect string.   It's correct, and I can certainly live with it, but wondering if there's a more elegant way to redirect to an external url with GET and param